Source/relay/TourGuide/Items of the Month/2022/Cosmic Bowling Ball.ash

```diff
@@ -2,59 +2,66 @@
 RegisterTaskGenerationFunction("IOTMCosmicBowlingBallGenerateTasks");
 void IOTMCosmicBowlingBallGenerateTasks(ChecklistEntry [int] task_entries, ChecklistEntry [int] optional_task_entries, ChecklistEntry [int] future_task_entries)
 {
+	if (!__iotms_usable[lookupItem("cosmic bowling ball")])
```

Comment:
stacey, does this actually work? feels like it might not given the inherent issues w/ the temporary qualities of the CBB.